src/auth/auth.controller.ts

```diff
@@ -1,4 +1,26 @@
-import { Controller } from '@nestjs/common';
+import { Body, Controller, Get, Post, Res, UseGuards } from '@nestjs/common';
+import { AuthService } from './auth.service';
+import { Response } from 'express';
+import { AuthLoginDto } from './dto/auth-login.dto';
+import { AuthGuard } from '@nestjs/passport';
+import { UserObj } from '../decorators/portal-users.decorator';
+import { Student } from '../student/student.entity';
 
 @Controller('auth')
-export class AuthController {}
+export class AuthController {
+   constructor(private readonly authService: AuthService) {}
+
+   @Post('/login')
+   async phoneRegister(
```

Comment:
this function will not be named 'phoneRegister'  :)  call it just 'login' 👍

Reply:
fixed in newest commit ;)